exporter/metric/metric.go

```diff
@@ -367,6 +367,23 @@ func recordToTypedValueAndTimestamp(r *export.Record) (*monitoringpb.TypedValue,
 
 	// TODO: Ignoring the case for Min, Max and Distribution to simply
 	// the first implementation.
+	//
+	// Currently the selector used in the integrator is `simple.NewWithExactMeasure`
```

Comment:
Looks like the function has been renamed to `NewWithExactDistribution` in v0.6.0 (based on your changes in #34) so the comment will need to be updated accordingly.

Reply:
Thank you for catching this. Updated!